src/ui/Omnibox/Omnibox.ts

```diff
@@ -387,11 +387,8 @@ export class Omnibox extends Component {
     }
 
     if (!_.isEmpty(query)) {
-      if (!this.options.enableQuerySyntax) {
-        data.queryBuilder.expression.add("<@- " + query + " -@>");
-      } else {
-        data.queryBuilder.expression.add(query);
-      }
+      data.queryBuilder.disableQuerySyntax = !this.options.enableQuerySyntax;

```

Comment:
The option's called that way in the search api https://developers.coveo.com/display/public/SearchREST/Query+Parameters#QueryParameters-disableQuerySyntax